src/super_context.ml

```diff
@@ -901,6 +902,24 @@ module PP = struct
           mli) in
     { m with impl ; intf }
 
+  let uses_ppx_driver ~pps =
+    match Option.map ~f:Pp.to_string (List.last pps) with
+    | Some "ppx_driver.runner" -> true
+    | Some _ | None -> false
+
+  let promote_correction ~uses_ppx_driver fn build =
+    if not uses_ppx_driver then
```

Comment:
Small nit: I find else clauses in `if (not ...)` confusing and prefer `if not x then foo else bar` to turn to `if x then bar else foo`.

Reply:
Personally, I find if/match expressions easier to read when the simple choice is first. That's why I wrote it this way